Simplenote/src/androidTest/java/com/automattic/simplenote/screenshots/ScreenshotTest.java

```diff
@@ -103,17 +110,40 @@ public void screenshotTest() throws InterruptedException {
         // app changing root activity (from login to notes list) but I haven't had the time to
         // research it properly
         Thread.sleep(2000);
-        waitForViewMatching(allOf(withId(R.id.note_title), withText(NOTE_TITLE)), 5000);
+        dismissVerifyEmailScreenIfNeeded();
+        // We want the full list of notes to be on screen when taking the screenshots, so let's wait
+        // for enough notes to be on screen to be relatively confident that's happened.
+        //
+        // Obviously, it would be better to have something like "wait for notes to load" but I
+        // wasn't able to find a way to achieve this – Gio
+        List<String> noteTitles = Arrays.asList(
+                NOTE_FOR_EDITOR_SHOT_TITLE,
+                "Bret Victor's Quote Collection",
+                "Back on track",
+                NOTE_FOR_INTERLINKING_SHOT_TITLE
+        );
+        for (String title:noteTitles) {
+            waitForViewMatching(allOf(withId(R.id.note_title), withText(title)), 5000);
+        }
```

Comment:
This seems to be a bit more robust that just waiting for one individual note.

Reply:
I like this approach, but I wonder if there's a way to construct a selector that waits for all of these in parallel? That said, the test's aren't slow, so probably not worth optimizing for at this point.